exercise/node-process-argv/echo.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable no-console */
+// this is a message to see what I can see 
+console.log("I can see this messages clear as day!!!!")
```

Comment:
`echo.js` should print back what the user types, so doing this:

```
node echo.js 'hello, michael'
```

should print back to the terminal

```
hello, michael
```

Reply:
OK I'll do that 
thank you